Update localization.rst (#3895)
mattermost/docs: source/developer/localization.rst

4. Go to **Groups**.
5. Add the user to ``mattermost@TrustedReviewers`` group.
6. Hit **Save**.

> To add a new language to the WIP project:

Reviewer: ```suggestion
To add a new language:
```

> 1. Go to https://translate.mattermost.com/projects/i18n-wip/ .

Reviewer 2: ```suggestion
1. Go to `https://translate.mattermost.com/projects/i18n-wip/<https://translate.mattermost.com/projects/i18n-wip/>`_.
```

> 1. Go to `https://translate.mattermost.com/projects/i18n-wip/<https://translate.mattermost.com/projects/i18n-wip/>`_.

Reviewer 2: ```suggestion
1. Go to `https://translate.mattermost.com/projects/i18n-wip/<https://translate.mattermost.com/projects/i18n-wip>`__.
```

> 2. Click on any of the three components **Tools > Start new translation** and add the languages you want to add and click **Save**.

Reviewer 2: Which three components are there? Or can the user just click "Tools"?

Author: No, the user has to click one component.

> 2. Click on any of the three components **Tools > Start new translation** and add the languages you want to add and click **Save**.

Reviewer: ```suggestion
2. Click any of the components on the resulting page, then go to **Tools > Start new translation**. Select the languages you want to add, then click **Start new translation**.
```

> 3. Repeat for the rest of the components or it could happen that Weblate already creates the language for the components in the same project.

Reviewer 2: If Weblate already creates the language does that mean the language was already there and not listed? Or is this a step to update the language in the other components?

Author: This is a step to create the language for other components. Sometimes it is there already, sometimes not. It is not made clear by weblate why.

> 3. Repeat for the rest of the components or it could happen that Weblate already creates the language for the components in the same project.

Reviewer: ```suggestion
3. Repeat step 2 for every component in `https://translate.mattermost.com/projects/i18n-wip/<https://translate.mattermost.com/projects/i18n-wip/>`
```

> 3. Repeat step 2 for every component in `https://translate.mattermost.com/projects/i18n-wip/<https://translate.mattermost.com/projects/i18n-wip/>`

Reviewer 2: ```suggestion
3. Repeat step 2 for every component in `https://translate.mattermost.com/projects/i18n-wip/<https://translate.mattermost.com/projects/i18n-wip/>`_.
```
